src/parser/sparqlParser/SparqlQleverVisitor.h

```diff
@@ -199,52 +200,53 @@ class SparqlQleverVisitor {
   [[nodiscard]] std::optional<parsedQuery::Values> visit(
       Parser::ValuesClauseContext* ctx);
 
-  ParsedQuery visit(Parser::UpdateContext* ctx);
+  [[nodiscard]] ParsedQuery visit(Parser::UpdateContext* ctx);
```

Comment:
Don't use [[nodiscard]] without a reason, it clutters the code. You can deactivate that clang-tidy check in your local IDE.